packages/ui5-config/src/ui5config.ts

```diff
@@ -413,6 +409,62 @@ export class UI5Config {
         return this;
     }
 
+    /**
+     * Returns the serve static config.
+     *
+     * @param hasfioriToolProxy - whether the config contains a fiori-tools-proxy middleware
+     * @param paths - serve static paths for the reuse libraries
+     * @returns the serve static middleware config
+     */
+    private getServeStaticConfig(
+        hasfioriToolProxy: boolean,
+        paths: ServeStaticPath[]
+    ): CustomMiddleware<{ paths: ServeStaticPath[] }> {
+        return hasfioriToolProxy
```

Comment:
```suggestion
        return {
            name: serveStatic,
                  beforeMiddleware: addFioriToosProxy ? fioriToolsProxy : 'compression',
                  configuration: {
                      paths: paths
                  }
        }
```

Reply:
I didn't make this change, in the case `addFioriToosProxy` is true the config is : 

```
beforeMiddleware: fioriToolsProxy
```

if `addFioriToosProxy` is false the config is : 
```
afterMiddleware: 'compression'
```

Reply:
Sorry my mistake, thx @cianmSAP